src/lib/ngsi/Scope.cpp

```diff
@@ -308,6 +306,46 @@ int Scope::fill
     return -1;
   }
 
+  //pointVectorRelease(pointV); ??
```

Comment:
Looks good, excpeto for one thing - the call to pointV.clear() on line 310 seems unnecessary. Not that it hurts but ...
The idea is to 'inherit the points from the vector **pointV** to the geometry objects.
After inheriting, the points cannot be freed (pointVectorRelease cannot be called), but the original vector, pointV, must be destroyed.
In case of errors, the entire vector with its points must be destroyed, so pointVectorRelease must be called apart from destroying the vector pointV itself.

Also, the comment on line 309 should be removed.
We **cannot** call pointVectorRelease, as that would free all the points stored in the geometry objects.